src/twisted/tap/ftp.py

```diff
@@ -58,6 +69,8 @@ def makeService(config):
     f.userAnonymous = config["userAnonymous"]
     f.portal = p
     f.protocol = ftp.FTP
+    if config.get("passivePortRange"):
```

Comment:
I don't have much experien with the twisted application framework.

What happen if the `--passive-port-range` is not passed?

Does it returns None? emtpy range ? False ?

I think is best to have a test and an explicit check for the value.

```suggestion
    if config.get("passivePortRange") is not None:
```

---------

I see that for a missing port, there is a KeyError exception handling and a fallback to a default value.

Just asking. Wny not have something similar here?

Reply:
> I don't have much experien with the twisted application framework.

[`twisted.python.usage.Options` subclasses `typing.Dict`](https://github.com/starrify/twisted/blob/ef14e5f2b31e8a5d6a7c8f83bc15ec22f25fbd0d/src/twisted/python/usage.py#L66). I hope this answers the following lines of questions as well.

> Wny not have something similar here?

It's not a not. I would assume that you assumed that it does not fall back to the default behavior upon missing to provide the argument. It does.